ArcGISRuntimeSDKQt_CppSamples/Routing/FindServiceAreasForMultipleFacilities/FindServiceAreasForMultipleFacilities.cpp

```diff
@@ -45,6 +44,7 @@
 #include "Envelope.h"
 #include "Polygon.h"
 #include "Graphic.h"
+#include "ErrorException.h"
 
 #include <QUuid>
 #include <QUrl>
```

Comment:
Is QFuture already included somewhere? I would think you need it in this file.

Reply:
QFuture is included in the header file.